project_task_add_very_high/__manifest__.py

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Why did you remove the Copyright from the header?

Reply:
I believed that the Copyright should be only in the "author" tag
This is one of the small changes I will do in a few days.